browser/brave_wallet/brave_wallet_provider_delegate_impl.cc

```diff
@@ -19,4 +51,45 @@ void BraveWalletProviderDelegateImpl::ShowConnectToSiteUI() {
   brave::ShowWalletBubble(browser);
 }
 
+void BraveWalletProviderDelegateImpl::RequestEthereumPermissions(
```

Comment:
I think there might be a security issue here because this is attached to the webcontents, but we need to know the render frame that initiated the request or we'll be displaying the wrong url for the permission

Reply:
Fixed in https://github.com/brave/brave-core/pull/9443/commits/fc9e5ed002f21d8e60558f1650082590b5c879ea